mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/type/impl/BytesAsBsonBinaryType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.type.impl;
+
+import org.hibernate.type.BinaryType;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+public class BytesAsBsonBinaryType extends BinaryType {
+	public static final String TYPE_NAME = "bson_binary_bytes";
```

Comment:
Why did you create a new class for this? Use Binary type directly.

We already have many classes to map type and I would prefer to not add new ones unless needed